src/Auth0.OidcClient.Android/Auth0.OidcClient.Android.csproj

```diff
@@ -71,8 +71,8 @@
     <PackageReference Include="IdentityModel.OidcClient">
       <Version>3.1.2</Version>
     </PackageReference>
-    <PackageReference Include="Xamarin.Android.Support.CustomTabs">
-      <Version>28.0.0.3</Version>
+    <PackageReference Include="Xamarin.AndroidX.Browser">
```

Comment:
On android libraries, making this change is a breaking change. I couldn't find much about xamarin. But look at [this](https://github.com/xamarin/AndroidX#phases):

> Keep in mind that all of your app's dependencies must also be compiled against AndroidX bindings to use these.

Those using `auth0-oidc-client-net` on Android would need to have all their dependencies using AndroidX or enable the `jetifier` preprocessor (which is OFF by default).

Reply:
Probably best to pull the trigger and bump the major.

Reply:
This is now affecting us, we cannot upgrade and we need to.  Can we have an AndroidX version of the library in Nuget?